GitUI/Resources/ChangeLog.md

```diff
@@ -2,225 +2,225 @@
 Changelog
 =========
 ### Version 2.49.03 (26 March 2017)
-* Fixed issue #3605. NullReferenceException from CanBeGitUrl when trying to clone git repository from dashboard.
-* Fixed issue #3578. File history and blame not show when path to file contain cyrilic chars bug reproducible.
+* Fixed issue [#3605](//github.com/gitextensions/gitextensions/issues/3605). NullReferenceException from CanBeGitUrl when trying to clone git repository from dashboard.
```

Comment:
The omission of the URL scheme means the link adopts the scheme of the page on which it is displayed. Adding `https:` would work too, but it's fine without. You can [test it for yourself](https://github.com/drewnoakes/gitextensions/blob/f2f1fe91397869ad17b4b203cac580308a35fb95/GitUI/Resources/ChangeLog.md).